concordium-consensus/src/Concordium/MultiVersion.hs

```diff
@@ -1228,7 +1228,7 @@ startBaker MultiVersionRunner{mvBaker = Nothing, ..} =
         Runner
         LLError
         "Attempted to start baker thread, but consensus was started without baker credentials."
```

Comment:
I might be wrong, but won't this still be logged in the new consensus? The message seems only relevant for the old consensus.

Reply:
This should never happen because the Rust code only `startBaker` when there are baker credentials.